proto/proto-backplane-grpc/src/main/proto/deephaven/proto/console.proto

```diff
@@ -52,7 +65,7 @@ message StartConsoleRequest {
 message StartConsoleResponse {
     io.deephaven.proto.backplane.grpc.Ticket result_id = 1;
 
-    // TOOD i think these are unused without an init script thingie
+    // TODO i think these are unused without an init script thingie
```

Comment:
I don't think this applies any more once app mode is merged, since the user will instead be using a stream to get variables that already existed from the console? Or does the app+console mode not address this?